### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by KayLemba**
In this milestone I did the following:In the following milestone I have:
- Create table named owners.
- Create table named species.
- Modify animal's table.
- Insert data into the tables.
- Written SQL statements using the join keyword.
This was all done with the guidance of  the project requirements.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -11,4 +11,26 @@ CREATE TABLE animals (
...
+);
+
+/* Create species table */
+CREATE TABLE species (
+ id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+ name VARCHAR(50)
+);
+
+/* Modify primary key */
+
```
Comment (DammyShittu) [inline schema.sql]: Kindly remove the `species` column as the requirement states 👇

File: data.sql [hunk 2]
```diff
@@ -32,3 +32,24 @@ VALUES ('Boarmon',  '2005/06/07', '7', 'true', '20.4');
...
+INSERT INTO owners (full_name, age) VALUES ('Melody Pond', 77);
+INSERT INTO owners (full_name, age) VALUES ('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES ('Jodie Whittaker', 38);
+
+/* Insert Species Names Into Species Table. */
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
+/* Update animals Table. */
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Digimon') WHERE name LIKE '%mon';
+UPDATE animals SET species_id = (SELECT id FROM species WHERE name = 'Pokemon') WHERE name NOT LIKE '%mon';
```
Comment (DammyShittu) [inline data.sql:48]: [OPTIONAL]

- Great implementation! 👍
Here 👇 is another way you can `UPDATE` the `animals` table

```
 UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;
```

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @KayLemba , 

Great job so far! 👏 

There is a very little issue that you still need to work on to go to the next milestone but you are almost there!

## REQUIRED CHANGES ♻️

_Check the comment under the review._

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 2 above)
Comment (KayLemba) [reply data.sql:48]: Thanks for the correction @DammyShittu

File: schema.sql (hunk 1 above)
Comment (KayLemba) [reply schema.sql]: Hello @DammyShittu  Your comment doesn't seem to be clear. please shade more light

Comment (KayLemba) [reply schema.sql]: Thanks its been implemented

**Review by andersonlebon — APPROVED** [review]
Hi @KayLemba 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 :clap: :clap: :100: 

nodding-yes

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 

## Optional suggestions
- [ ] Kindly Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by KayLemba** [comment]
Thank you

**Pull request merged**